src/macros.rs

```diff
@@ -0,0 +1,44 @@
+/// Implements conversion traits on a type wrapping a three-rs type. Useful for when you wrap a
+/// three-rs type with your own struct. Allows you to use that struct in place of any three-rs
+/// object.
+///
+/// Implements the following traits:
+///
+/// * `AsRef<NodePointer>`
+/// * `Deref<Target=Object>`
+/// * `DerefMut<Object>`
+///
+/// ```
+/// // Implements conversion traits for MyStruct using field `internal_three_type`
+/// three_object_wrapper!(MyStruct#internal_three_type);
+/// // If field is omitted, it defaults to `object`
+/// three_object_wrapper!(MyStruct); // equivalent to three_object_wrapper!(MyStruct#object);
+/// ```
+#[macro_export]
+macro_rules! three_object_wrapper {
```

Comment:
I don't think it is. The reason I added it is because it's nice to know where macros come from. If you look at the code where I used this (linked in the PR), it's very clear that the macro being used is for this library and its types. 

I'm open to another name too if you have something in mind. :)